carsus/io/cmfgen/base.py

```diff
@@ -455,27 +455,47 @@ class CMFGENReader:
     ----------
     levels : DataFrame
     lines : DataFrame
+    collisions : DataFrame
 
     """
 
-    def __init__(self, data, priority=10):
+    def __init__(self, data, priority=10, collisions=False, temperature_grid=None):
         """
         Parameters
         ----------
         data : dict
             Dictionary containing one dictionary per species with 
             keys `levels` and `lines`.
-
+        collisions : bool
+            Option to calculate collisional data, by default False.
         priority: int, optional
             Priority of the current data source, by default 10.
+        temperature_grid : list of numbers, optional
```

Comment:
Does this have to be a `list` or rather an `array-like`?

Reply:
Done in [51dcf85](https://github.com/tardis-sn/carsus/pull/272/commits/51dcf858b0cf2ef84fc262a73fb9c3f3de5c7db4)